submissions/smillims/memory-pair-game/js/script.js

```diff
@@ -0,0 +1,138 @@
+'use strict';
+
+const cards = document.querySelector('.cards');
+
+let lockCard = false;
+let firstCard = '';
+let secondCard = '';
+
+const titleImages = [
+	
+	{	
+		'id': 'bentley',
+		'srcFront': 'img/logo.png',
```

Comment:
why you don't make `srcFront`  as global variable ? (It is same for all images)

Reply:
I thought that every separate object would be with their information about their block

Reply:
@smillims then I suggest you you use this approach :

```
const srcFront = 'img/logo.png',
...

{	
  id: 'bentley',
  srcFront,
}
```
or 
```
const SRC_FRONT = 'img/logo.png',
...

{	
  id: 'bentley',
  srcFront: SRC_FRONT,
}
```

P.S.
you don't need to write quotes in you object definition
```suggestion
		srcFront: 'img/logo.png',
```